packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -141,7 +141,8 @@ export const NewsFeedInput = React.forwardRef<
     const selectedNetwork = useSelectedNetworkInfo();
     const selectedNetworkId = selectedNetwork?.id || "teritori";
     const selectedWallet = useSelectedWallet();
-    const userId = getUserId(selectedNetworkId, selectedWallet?.address);
+    const posterId = daoId ? daoId : selectedWallet?.userId;
+    const posterAddress = parseUserId(posterId)[1];
```

Comment:
nit, the usual pattern to access hooks return is destructuring (hooks in official react docs are used like this)

```suggestion
    const [, posterAddress] = parseUserId(posterId);
```

Reply:
i don't use this line anymore since i parse it in the modal
but i note this to avoid repeating this pattern in next PRs